examples/integrations/itk/SelectROI.ipynb

```diff
@@ -32,12 +32,12 @@
     "# Install dependencies for this example\n",
```

Comment:
<p>Looks like we lost the embedded screenshot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/itkwidgets/pull/715/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Good catch!